server/src/main/scala/zioslick/ZioSlickSupport.scala

```diff
@@ -23,7 +23,7 @@ import slick.dbio.DBIO
 import zio.ZIO
 
 trait ZioSlickSupport {
-  def fromDBIO[R](dbio: DBIO[R]): SlickZIO[R] =
+  def fromDBIO[R](dbio: DBIO[R]): ZIO[DatabaseProvider, RepositoryException, R] =
```

Comment:
In ZIO you use type aliases all over the place for one very good reason: you end up with brittle code if you specify all of the dependencies all over the place. If you use type aliases you can add/remove dependencies easily and only the user of the dependency and the provider need to know about it, with no middle code changing. 
For example, imagine that we want to add a Logger to the environment.
The way I particularly use this is that I have a type alias for my whole application.

Reply:
I reverted the change